package.json

```diff
@@ -84,6 +84,7 @@
     "ember-ace": "^1.2.0",
     "ember-bootstrap-datepicker": "^2.0.3",
     "ember-cli-babel": "6.4.1",
+    "ember-concurrency": "0.8.12",
```

Comment:
Did you mean to have `ember-concurrency` twice here in the same section?

Reply:
nope! fixed.